osutil/disks/disks.go

```diff
@@ -336,59 +337,88 @@ func (d *disk) FindMatchingPartitionUUID(label string) (string, error) {
 	// if we haven't found the partitions for this disk yet, do that now
```

Comment:
It seems better to return "filepath.EvalSymlinks(filepath.Join("/dev/block", d.Dev()))" which then can be directly used for do the systemd mount, without needing to join it with `/dev/disk/by-partuuid` to create a symlink to the block device one wants.

Reply:
yes this is the most correct, least racy thing to do that we will switch to doing eventually, but since we want this PR for 1.0, I will take it as a followup to this rather than address this in the PR